scapy/layers/inet.py

```diff
@@ -1063,9 +1062,9 @@ def __init__(self, res=None, name="Traceroute", stats=None):
         self.nloc = None
 
     def show(self):
-        return self.make_table(lambda (s,r): (s.sprintf("%IP.dst%:{TCP:tcp%ir,TCP.dport%}{UDP:udp%ir,UDP.dport%}{ICMP:ICMP}"),
-                                              s.ttl,
-                                              r.sprintf("%-15s,IP.src% {TCP:%TCP.flags%}{ICMP:%ir,ICMP.type%}")))
+        return self.make_table(lambda s_r: (s_r[0].sprintf("%IP.dst%:{TCP:tcp%ir,TCP.dport%}{UDP:udp%ir,UDP.dport%}{ICMP:ICMP}"),
```

Comment:
Just to be clear. On python 3, contrary to Python 2, tuples work like so:

- If we try without tuples:
```
Python 3.6.0 (v3.6.0:41df79263a11, Dec 23 2016, 08:06:12) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> def test(_tuple):
...     a = lambda s, r: print(s + ";" + r)
...     a(_tuple)
...
>>> test("1", "2")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: test() takes 1 positional argument but 2 were given
```

- If we apply Python 2's tuples:
```
Python 3.6.0 (v3.6.0:41df79263a11, Dec 23 2016, 08:06:12) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> def test(_tuple):
...     a = lambda s,r: print(s + ";" + r[1])
...     a(_tuple)
...
>>> test(("1", "2"))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<stdin>", line 3, in test
TypeError: <lambda>() missing 1 required positional argument: 'r'
```

Which means that we need to use:
```
>>> def test(_tuple):
...     a = lambda s_r: print(s_r[0] + ";" + s_r[1])
...     a(_tuple)
...
>>> test(("1", "2"))
```